src/pages/home/report/ReportActionCompose/index.js

```diff
@@ -741,214 +742,220 @@ class ReportActionCompose extends React.Component {
+                        )}
+                        <View
+                            style={[styles.justifyContentEnd]}
+
+                            // Keep focus on the composer when Send message is clicked.
+                            onMouseDown={e => e.preventDefault()}
+                        >
+                            <Tooltip text={this.props.translate('common.send')}>
+                                <TouchableOpacity
+                                    style={[styles.chatItemSubmitButton,
+                                        (this.state.isCommentEmpty || hasExceededMaxCommentLength) ? undefined : styles.buttonSuccess,
+                                    ]}
+                                    onPress={this.submitForm}
+                                    disabled={this.state.isCommentEmpty || isBlockedFromConcierge || this.props.disabled || hasExceededMaxCommentLength}
+                                    hitSlop={{
+                                        top: 3, right: 3, bottom: 3, left: 3,
+                                    }}
+                                >
+                                    <Icon src={Expensicons.Send} fill={(this.state.isCommentEmpty || hasExceededMaxCommentLength) ? themeColors.icon : themeColors.textLight} />
+                                </TouchableOpacity>
+                            </Tooltip>
+                        </View>
                     </View>
-                </View>
-                <View style={[
-                    styles.flexRow,
-                    styles.justifyContentBetween,
-                    styles.alignItemsCenter,
-                    (!this.props.isSmallScreenWidth || (this.props.isSmallScreenWidth && !this.props.network.isOffline)) && styles.chatItemComposeSecondaryRow]}
-                >
-                    {!this.props.isSmallScreenWidth && <OfflineIndicator containerStyles={[styles.chatItemComposeSecondaryRow]} />}
-                    <ReportTypingIndicator reportID={this.props.reportID} />
-                    <ExceededCommentLength comment={this.comment} onExceededMaxCommentLength={this.setExceededMaxCommentLength} />
-                </View>
+                    <View style={[
+                        styles.flexRow,
+                        styles.justifyContentBetween,
+                        styles.alignItemsCenter,
+                        (!this.props.isSmallScreenWidth || (this.props.isSmallScreenWidth && !this.props.network.isOffline)) && styles.chatItemComposeSecondaryRow]}
+                    >
+                        {!this.props.isSmallScreenWidth && <OfflineIndicator containerStyles={[styles.chatItemComposeSecondaryRow]} />}
+                        <ReportTypingIndicator reportID={this.props.reportID} />
+                        <ExceededCommentLength comment={this.comment} onExceededMaxCommentLength={this.setExceededMaxCommentLength} />
+                    </View>
+                </OfflineWithFeedback>
```

Comment:
Is there any reason to keep `{this.state.isDraggingOver && <ReportDropUI />}` outside `OfflineWithFeedback`? Just wanted to make sure it was intentional change. So, we won't have any regression in the future.

Reply:
@sobitneupane It doesn't matter if it's outside or inside `offlineWithFeedback` because we are rendering it with a portal anyway. I tested it both ways and it works. But I think it's better this way because only the essential components are wrapped with `offlineWithFeedback`